package.json

```diff
@@ -30,84 +30,67 @@
     "dev": "node build/dev-server.js",
     "docs": "node build/build.js",
     "build": "npm run build:umd",
```

Comment:
I remove this because It was bugged and because in seems to not be used.  
On `module` config the file used is the one generated by the `build:umd` script (`dist/vue-cookie-law.js`).